Documentation/networking/overriding-defaults.md

```diff
@@ -0,0 +1,41 @@
+# Overriding defaults
+
+This document holds information about modifying or replacing builtin defaults and is only recommended to advanced users.
+Please make sure to have read the [networking overview page](overview.md) before treading into these things.
+
+## Overriding the "default" network
+
+If a network has a name "default", it will override the default network added by rkt.
+It is strongly recommended that such network also has type "ptp" as it protects from the pod spoofing its IP address and defeating identity management provided by the metadata service.
+
+## Overriding network settings
+
+The network backend CNI allows the passing of [arguments as plugin parameters](https://github.com/appc/cni/blob/master/SPEC.md#parameters), specifically `CNI_ARGS`, at runtime.
+These arguments can be used to reconfigure a network without changing the configuration file.
+rkt supports the `CNI_ARGS` variable through the command line argument `--net`.
+
+### Syntax
+
+The syntax for passing arguments to a network looks like `--net="$networkname1:$arg1=$val1;$arg2=val2"`.
+The usage of double quotes is mandatory due to the `;` being used as separator within the arguments for a single network.
+To allow the passing of arguments to different networks simply append the arguments to the network name with a colon (`:`), and separate the arguments by semicolon (`;`).
+All arguments can either be given in a single instance of the `--net`, or can be spread across multiple uses of `--net`.
+*Reminder:* the separator for the networks (and their arguments) within one `--net` instance is the comma `,`.
+A network name must not be passed more than once, not within the same nor throughout multiple instances of `--net`.
+
```

Comment:
I don't know, I like to have both easy examples that can be copy/pasted and bigger ones to have a better idea. But that's a detail, you can skip this if you think it is not necessary.